examples/imagenet/inception/imagenet_distributed_train_pipeline.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Shouldn't we have copyright for pipeline examples?

Reply:
Since I left the copyright for the `imagenet_distributed_train.py` file, I did the same for this one as well (even though we've modified it from the original).